spec/directions/bake_numbered_exercise/v1_spec.rb

```diff
@@ -63,4 +63,50 @@
       )
     end
   end
+
+  context 'when even solutions are suppressed' do
+    context 'when number is odd' do
+      it 'works' do
+        described_class.new.bake(exercise: exercise1, number: 1, suppress_even_solution: true)
```

Comment:
In this context I have changed the number value to 1 (integer) instead of 1.1 (it is recognized as float), because of the error I've got while this test was running: 
![image](https://user-images.githubusercontent.com/55203598/119127465-a852da80-ba34-11eb-8824-c917c23c64e2.png)
The methods `odd?, even?` can't be used on such numbers.
I decided to not change the lines I have added from line 13 in `bake_numbered_exercise.rb` file after this error and change the test data instead because the only books that currently utilize suppressing half of the solutions are History and Sociology and there the numbering of the exercises is created to contain only one number (1). These books don't have exercises in notes that are numbered using two counters creating one number (1.1) like Calculus for example. Also the numbers for exercises in notes are created in separate files (f.ex `bake_checkpoint.rb` for checkpoint notes for Calculus recipe) and I wasn't sure If we will ever need to suppress half of the the solutions from exercises in notes. But let me know what do you think about it?

Reply:
Interested in seeing your recipe to see how I can help here